controllers/allcontrollers.go

```diff
@@ -208,7 +200,7 @@ func initRoutemapCMInformer(stopCh <-chan struct{}, config *Config, client k8scl
 
 // Start starts all Iter8 controllers if this pod is the leader
 func Start(stopCh <-chan struct{}, client k8sclient.Interface) error {
-	config, err := readConfig()
+	config, err := ReadConfig()
```

Comment:
Let us make `config` an externally visible object, and `readConfig` and internal method.

Reply:
Should also require persistence for A/B/n testing. Don't need boolean config. If comfig needed should be separate configmap (file).